### PR TITLE
allow relative redirects

### LINKED ISSUE #4: Allow relative Redirects
The redirects after sending should also allow relative urls. The request referrer could be used as the host.

Example:

- called from https://example.com/contact
  - redirect to `/mailsuccess`
- mailform should be redirect to https://example.com/mailsuccess

I have written a version which allows both, absolute and relative URLs and will be create a PR

### PR DESCRIPTION
**Pull request by joe128**
fixes #4

### CODE REVIEW

File: src/router.ts [hunk 1]
```diff
@@ -98,19 +98,30 @@ router.post("/:target", async (req: Request, res: Response) => {
...
+    let redirectUrl = targetRedirectUrl
+
+    const urlPattern = /^(https?):\/\//;
+    if(!urlPattern.test(redirectUrl)) {
+        redirectUrl = (req.header('Referer') || '/') + redirectUrl;
+    }
+
+    return redirectUrl
+}
+
```
Comment (Feuerhamster) [inline src/router.ts]: Please move that to a separate file. The router should only contain express routes, not utility functions.

**Review by Feuerhamster — CHANGES_REQUESTED** [review]

File: src/router.ts (hunk 1 above)
Comment (joe128) [reply src/router.ts]: OK, i will fix it tomorrow. do you prefer 'util/redirect.ts' or Just 'util.ts' in Root?

Comment (Feuerhamster) [reply src/router.ts]: > OK, i will fix it tomorrow. do you prefer 'util/redirect.ts' or Just 'util.ts' in Root?

'utils/redirect.ts' is fine. Thank you :)

Comment (joe128) [reply src/router.ts]: getRedirectUrl is now extracted to util/redirect.ts

**Review by Feuerhamster — APPROVED** [review]

**Pull request merged**